app/controllers/api/v0/swagger/models/highlight.rb

```diff
@@ -42,25 +42,84 @@ module Api::V0::Swagger::Models::Highlight
     end
   end
 
+  swagger_schema :GetHighlights do
+    key :required, [:source_type]
+    property :page do
+      key :type, :integer
+      key :description, 'The page number of paginated results, one-indexed. Defaults to 1.'
+    end
+    property :per_page do
+      key :type, :integer
+      key :description, 'The number of highlights per page for paginated results. Defaults to 15.'
+    end
+    property :order do
+      key :type, :string
+      key :enum, %w[asc desc]
+      key :description, 'The desired sort order on the highlight creation time. Defaults to asc.'
+    end
+  end
+
   swagger_schema :Highlight do
     key :required, [:id]
   end
 
+  swagger_schema :Highlights do
+    # organization from https://jsonapi.org/
+    property :meta do
+      property :page do
+        key :type, :integer
+        key :description, 'The response page number'
+      end
+      property :per_page do
+        key :type, :integer
+        key :description, 'The response per page'
+      end
+      property :total_count do
+        key :type, :integer
+        key :description, 'The number of returned highlights'
```

Comment:
think this is intended to be the number of total results, not just those in this page of results (which are what is "returned")